### DISCUSSION TITLE
How to compile messages to .js format instead of .mjs

### DISCUSSION THREAD

**Discussion by SelinaMT** [lingui/js-lingui#1252] [Q&A] [answered]
Hi, I updated lingui from 2.9.2-version to 3.13.3. 
In this change compiled (with lingui compile) message format changed from messages.js to messages.mjs. I have some trouble with this format in my react application. Is it still possible to compile messages to .js format, and if it is, how can I configure this?

**Comment by semoal** [comment, marked as answer]
Check your configuration if you're using `compileNamespace: es` that exports to esm modules, which must be as `.mjs`, instead of that you can use `cjs` or `ts` if you're using TypeScript
https://lingui.js.org/ref/conf.html#compilenamespace

**Reply by SelinaMT** [reply]
Thank you, this solved it!
